sdk/communication/communication-phone-numbers/test/public/lro.update.spec.ts

```diff
@@ -37,15 +38,30 @@ matrix([[true, false]], async function(useAad) {
     });
 
     it("can update a phone number's capabilities", async function() {
-      const update: PhoneNumberCapabilitiesRequest = { calling: "none", sms: "outbound" };
       const updatePoller = await client.beginUpdatePhoneNumberCapabilities(
         purchasedPhoneNumber,
         update
       );
 
-      const phoneNumber = await updatePoller.pollUntilDone();
+      // TODO: this validation is flakey because multiple tests attempt to update the same number
+      // re-enable when we make each lang run it's own number
+      // const phoneNumber = await updatePoller.pollUntilDone();
+      await updatePoller.pollUntilDone();
       assert.ok(updatePoller.getOperationState().isCompleted);
-      assert.deepEqual(phoneNumber.capabilities, update);
+      // assert.deepEqual(phoneNumber.capabilities, update);
     }).timeout(60000);
+
+    it("throws on invalid update request", async function() {
```

Comment:
We can make this a bit more specific, there are multiple ways how an update could be invalid.

```suggestion
    it("update throws when phone number isn't owned", async function() {
```